HW_15/test_human.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+
+class TestHuman:
+
+
+    @pytest.mark.positive
+    def test_increase_age(self, create_human):
+        human = create_human
+        human.grow()
+        actual_age = human.age
+        expected_age = 26
+        assert actual_age == expected_age, "The human age is not increased."
+
+    @pytest.mark.negative
+    def test_increase_age_over_limit(self, create_human_over_age_limit):
+        human = create_human_over_age_limit
+        human.grow()
+        actual_age = human.age
+        expected_age = 100
+        assert actual_age == expected_age, "The human age exceeds the limit."
+
+    @pytest.mark.negative
+    def test_increase_dead_human_age(self, create_dead_human):
+        human = create_dead_human
+        with pytest.raises(Exception):
+            human.grow(), "The age of the dead human is increased."
+
+    @pytest.mark.positive
+    def test_change_gender(self, create_human):

```

Comment:
you can parametrize this test. 
or randomize, some times it will change gender to male another time to female